libs/src/services/ABIDecoder/AudiusABIDecoder.ts

```diff
@@ -63,4 +65,33 @@ export class AudiusABIDecoder {
   static decodeLogs(_: string, logs: Log[]) {
     return abiDecoder.decodeLogs(logs)
   }
+
+  static decodeAbi (encodedABI: string): Map<string, string> {
+    const decodedABI = AudiusABIDecoder.decodeMethod('EntityManager', encodedABI)
```

Comment:
This seems generally more useful than the above stuff as long as the consumer is able to pass in the ABI. I think the need for your change here also may explain why CI sometimes fails in finding these.

https://app.circleci.com/pipelines/github/AudiusProject/audius-protocol/39580/workflows/779cd4d2-517d-45bc-b5c9-13857ad8efc6/jobs/510721?invite=true#step-106-911

Reply:
ah gotcha